src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    &:hover {
+      // transform: translateY(-20px);
+      transform: scale(1.2);
+      color: #34568b;
+      transition: transform 300ms, color 300ms, scale 300ms;
```

transform covers scale transition, no need to mention``
```suggestion
      transition: transform 300ms, color 300ms;
```